src/proxy_handler.cc

```diff
@@ -0,0 +1,180 @@
+#include "proxy_handler.h"
+
+#include <string>
+
+RequestHandler::Status ProxyHandler::Init(const std::string& uri_prefix,
+    const NginxConfig& config)
+{
+  uri_prefix_ = uri_prefix;
+
+  int server_location = 0;
+  if (config.statements_[0]->tokens_[0] == "remote_port") {
+    server_location = 1;
```

Comment:
It indicates whether the server or port comes first within the ProxyHandler's configuration block. I've added a comment to explain this piece of code. Thanks!